ext/mysqli/tests/bug73462.phpt

```diff
@@ -1,17 +1,10 @@
 --TEST--
 Bug #73462 (Persistent connections don't set $connect_errno)
 --EXTENSIONS--
-mysqli
+all
```

Comment:
This is the `--EXTENSIONS--` section.

Marking as conflicting with `all` would only solve the problem if there is another test not properly marked as conflicting with `mysql`. Possibly, there are other connections (not caused by our tests) involved anyway?

Reply:
Oh, I had a complete brain fart. 🙂 I'm not sure this makes sense at all.